src/modules/countlikes.js

```diff
@@ -0,0 +1,13 @@
+const countLike = (likesList, itemId) => {
```

Comment:
- [ ] [Optional] The user from one session can register multiple likes on one item, please consider, fixing this you can disable the button after the user click so it will send just one like. This is not a requirement but it's highly recommended